task/rsync/0.1/Dockerfile

```diff
@@ -0,0 +1,13 @@
+ARG alpine_ver=3.13
+FROM alpine:${alpine_ver}.5
```

Comment:
maybe you just want [3.13](https://hub.docker.com/layers/alpine/library/alpine/3.13/images/sha256-9663906b1c3bf891618ebcac857961531357525b25493ef717bca0f86f581ad6?context=explore) (or alpine_ver) to get all the minor release  and have ARG working

Reply:
fixed.